packages/debugger-extension/schema/main.json

```
@@ -40,7 +40,7 @@
       "selector": "body"
     },
     {
-      "command": "debugger:continue",
+      "command": "debugger:pause-continue",
```

```suggestion
      "command": "debugger:continue",
```

This is highlighted by the failing test.

---

Thanks @fcollonval